docs/can-guides/commitment/recipes/file-navigator/file-navigator-advanced-3.js

```diff
@@ -1,3 +1,5 @@
+import { fixture, DefineMap } from "//unpkg.com/can@5/core.mjs";
```

Comment:
I think it’d be better to keep these alphabetical, i.e. `DefineMap, fixture`. This applies later in the guide too.